openedx/core/djangoapps/course_groups/views.py

```diff
@@ -362,3 +406,41 @@ def debug_cohort_mgmt(request, course_key_string):
         kwargs={'course_key': course_key.to_deprecated_string()}
     )}
     return render_to_response('/course_groups/debug.html', context)
+
+
+@expect_json
+@login_required
+def cohort_discussion_topics(request, course_key_string):
+    """
+    The handler for cohort discussion categories requests.
+    This will raise 404 if user is not staff.
+
+    Returns the JSON representation of discussion topics w.r.t categories for the course.
```

Comment:
Can you add an example of the return value? For example, see: https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/data_conversion.py#L10-L63